src/lib/common/sol-log-internal.h

```diff
@@ -36,7 +36,7 @@
 
 #ifdef SOL_LOG_ENABLED
 #define SOL_LOG_INTERNAL_DECLARE(_var, _name)    \
-    struct sol_log_domain _var = {               \
+    SOL_ATTR_UNUSED struct sol_log_domain _var = { \
```

Comment:
yep, we are declaring log on generated code that can be not used. This was happening with many samples.